backend/recipes/base.py

```diff
@@ -240,180 +245,46 @@ def runStep(self):
         if('icon' in step):
             self.icon = step['icon']
 
-        if 'baseTask' in step and step['baseTask'] != 'humanTask':
-            if ('task' in step):
-                task = step['task']
-                base = False
-            else:
-                task = step['baseTask']
-                base = True
-
-            if celery.runTask(task, step['parameters'], base):
-                self.status = 'running'
-            else:
-                self.status = 'error'
-                message = 'Internal error. Task already running.'
-                return False
-
-        if('parameters' in step) and ('time' in step['parameters']):
-            duration = timedelta(seconds=step['parameters']['time'])
-            self.stepCompletionTime = (datetime.now(tz=timezone.utc) + duration).isoformat()
-        else:
-            self.stepCompletionTime = None
+        if 'tasks' not in step and 'baseTask' in step and step['baseTask'] != 'humanTask':
```

Comment:
The only major feedback I can think of here is should 'baseTask', 'tasks', and other similar strings be constants, especially if this needs to be referenced anywhere else.